packages/trader/src/AppV2/Routes/routes.tsx

```diff
@@ -0,0 +1,33 @@
+import { lazy } from 'react';
+import { RouteProps } from 'react-router';
+import { routes } from '@deriv/shared';
+import Trade from 'AppV2/Containers/Trade';
+import Markets from 'AppV2/Containers/Markets';
+import Positions from 'AppV2/Containers/Positions';
+import Menu from 'AppV2/Containers/Menu';
+
+const traderRoutes: RouteProps[] = [
+    {
+        path: routes.trade,
+        component: Trade,
+        exact: true,
+    },
+    {
+        path: routes.markets,
+        component: Markets,
+    },
+    {
+        path: routes.trader_positions,
+        component: Positions,
+    },
```

Comment:
do we need something like `is_authenticated: true` here? to hide positions when a user is logged-out? )

Reply:
Nice catch. Thanks @maryia-deriv. Let me add it now.

Reply:
I've added the authentication check. Please review it again 🙏